packages/flutter_app_badger/tizen/src/tizen_badge.cc

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "tizen_badge.h"
+
+#include "log.h"
+
+TizenBadge::~TizenBadge() {
+  int ret = badge_remove(app_id_);
+  if (ret != BADGE_ERROR_NONE) {
+    LOG_ERROR("badge_remove() failed with error %d.", ret);
+  }
+}
+
+bool TizenBadge::Initialize() {
+  int ret = app_get_id(&app_id_);
```

Comment:
`app_id_` is not freed in destructor which will cause memory leak.

Reply:
done